invio/umbrel-app.yml

```
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: invio
+category: finance
+name: Invio
+version: "1.0.0"
```

Match to the version of Docker Image
```suggestion
version: "1.7.0"
```